exercises/accumulate/.meta/hints.md

```diff
@@ -5,9 +5,12 @@ It is typical to call [#to_enum](http://ruby-doc.org/core-2.3.1/Object.html#meth
 Here is an additional test you could add:
 
 ```ruby
-def test_no_block_is_passed
+def test_accumulate_squares_when_no_block_is_passed
   skip
-  result = [1, 2, 3].accumulate
-  assert_instance_of Enumerator, result
+  enum = [1, 2, 3].accumulate
+  assert_instance_of Enumerator, enum
+
+  result = enum.each { |number| number * number }
```

Comment:
again, a more informative variable name would be helpful here.
`accumulated_result` perhaps?